charts/argo-cd/Chart.yaml

```diff
@@ -23,4 +23,5 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Changed]: Update Argo CD to v2.5.9"
+    - kind: fixed
+      description: Align changelog structure to show changelogs on Artifact Hub
```

Comment:
Let's try this structure.
ref: https://github.com/artifacthub/hub/blob/master/charts/artifact-hub/Chart.yaml#L82-L83

<img width="694" alt="image" src="https://user-images.githubusercontent.com/30188755/215335928-2d2ed202-22ed-41f2-8cd4-576bc18d4898.png">